docs/reference/meson-compatibility.rst

```diff
@@ -52,6 +52,10 @@ versions.
    populate the package license and license files from the ones
    declared via the ``project()`` call in ``meson.build``.
 
+.. option:: 1.9.0
+
+   Meson 1.9.0 or later is required to support building for ``iOS``.
```

Comment:
```suggestion
   Meson 1.9.0 or later is required to support building for iOS.
```
iOS is a product name thus I don't think it should be rendered in fixed-width font like code.